css/main.css

```diff
@@ -0,0 +1,303 @@
+
+#main-header {
+    position: fixed;
+    top: 0px;
+    width: 100%;
+    background: #FFFFFF;
+    height: 70px;
+    line-height: 70px;
+}
+
+#main-header button {
+    margin: 10px;
+    color: black;
+    box-sizing: border-box;
+    border: 1px solid #CEB9B9;
+    border-radius: 5px;
+    background: none;
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 0px 15px;
+    font-weight: bold;
+    cursor: pointer;
+    outline: none;
+    z-index: 1;
+    height: 50px;
+    /*    behavior: url(/bower_components/css3pie/PIE.htc);*/
+}
+
+.hamburger {
+    line-height: 45px;
+    font-size: 1.4em;
+}
+
+.cross {
+    width: 57px;
+    font-size: 3em;
+    line-height: 70px;
+}
+
+.menu {
+    top: 70px;
+    padding-left: 0px;
+    margin-top: 0px;
+    z-index: 1000000;
+    font-weight: bold;
+    font-size: 0.8em;
+    width: 100%;
+    background: #131313;
+    position: fixed;
+    text-align: center;
+}
+
+.menu ul {
+    margin: 0;
+    padding: 0!important;
+    list-style-type: none;
+    list-style-image: none;
+}
+
+.menu li {
+    display: block;
+    padding: 15px 0 15px 0;
+}
+
+.menu ul li a {
+    text-decoration: none;
+    text-align: center;
+    margin: 0px;
+    color: #fff;
+}
+
+.menu ul li a:hover {
+    color: #fff;
+    text-decoration: none;
+}
+
+.menu a {
+    text-decoration: none;
+    color: white;
+}
+
+.menu a:hover {
+    text-decoration: none;
+    color: white;
+}
+
+.cross {
+    display: none;
+}
+
+
+/*************** body elemnt ***************/
+
+.container {
+    margin-top: 160px;
+}
+
+.menu {
+    display: none;
+}
+
+.glyphicon-home {
+    color: white;
+    font-size: 1.5em;
+    margin-top: 5px;
+    margin: 0 auto;
+}
+
+
+/*	allscritpet*/
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    margin: 0px;
+    font-size: 3em;
+    color: #EB0951;
+}
+
+p,
+li {
+    color: #97866F;
+    padding: 0px 15px 0px 15px;
+}
+
+.pink {
+    color: #EB0951;
+}
+
+
+/* form script*/
+
+#searchForm {
+    display: none;
+}
+
+
+/*header script*/
+
+.img-center {
+    line-height: 10px;
+}
+
+.img-logo {}
+
+.img-logo h2 {
+    display: none;
+}
+
+.img-logo h1 {
+    margin-left: 20px;
+    font-stretch: extra-condensed;
+}
+
+.img-center {
+    background-color: #e4dace;
+    border: 1px solid #C3BDB5;
+    display: block;
+    height: 70px;
+    margin: 10px auto auto;
+    overflow: hidden;
+    text-align: left;
+}
+
+.img-center h2 {
+    color: #FDFAF8;
+}
+
+.img-center h1 {
+    color: #EFECE3;
+}
+
+.img-center h1,
+.img-center h2 {
+    line-height: 1em;
+    margin-top: 0;
+    text-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    -ms-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    /* IE 9 */
+    -webkit-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    /* Chrome, Safari, Opera */
+    transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    white-space: nowrap;
+}
+
+
+/*body element*/
+
+.column-1 {
+    margin-top: 80px;
```

Comment:
As i told you, it's rather bad to have this kind of style on a common selector which could be used for a column somewhere else.